### ISSUE TITLE
Support for XChaCha20 Poly1305

### ISSUE THREAD

**Issue by sergioxdev** [rweather/arduinolibs#67]
Hi Guys

congratulations for the excellent work you have done,
I wanted to know, if you have already planned to add XChaCha20 Poly1305

**Comment by rweather** [comment]
Hi - This is the first I have heard of XChaCha20. :-)  But it looks like a simple adjustment to the key setup procedure based on random rather than sequential nonces.  The IETF draft has expired though, so it looks like the standardisation process may have stalled.  But I'll take a look to see what the impact is.

**Comment by sergioxdev** [comment]
Yes indeed the IETF draft expired in 2019, I don't understand why the standardization process has stalled.
Yes, there are simple adjustments.
